versions.tf

```diff
@@ -19,7 +19,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.0"
+      version = "~> 5.0"
```

Comment:
We want to stick to using the >= notation here. The version will always be constrained by the main "module" i.e. the infra repos that will call this module